### ISSUE TITLE
Change Tour of Heroes tutorial in the first Angular project

### ISSUE THREAD

**Issue by Doc415** [TheCSharpAcademy/TCSA.V2#342]
Tour of Heroes tutorial is not consistent with new version of Angular, and it does not work even if you follow the instructions.
It should be changed with an alternative tutorial

**Comment by ukpagrace** [comment]
Hey @Doc415 
Do you have an alternative tutorial you followed that works?

**Comment by Doc415** [comment]
> Hey @Doc415 
> Do you have an alternative tutorial you followed that works?

Unfortunately I dont have any yet

**Comment by VocalNight** [comment]
The tutorial for angular now is related to a renting website, it can be found here:

https://angular.dev/tutorials/first-app

Solving this issue would mean changing not only the whole article, but the project repo too.

**Comment by Doc415** [comment]
Yes indeed. I had to follow the new web sites tutorial and applied what i learned to old project. I had to spent double time . Angular is uptading major revisions in every six months. It s backward competable but still hard for new comers. I had to skip Angular area because of this. Completing React projects made ut easier to understand Angular. I still suggest to update first Angular project with a stable , not version dependent one.
